cmd/composectl/cmd/ps.go

```diff
@@ -44,13 +46,15 @@ type (
 	AppStatus []ServiceStatus
 
 	psOptions struct {
-		Format string
+		Format       string
+		CheckInstall bool
 	}
 )
 
 func init() {
 	opts := psOptions{}
 	psCmd.Flags().StringVar(&opts.Format, "format", "table", "Format the output. Values: [table | json]")
+	psCmd.Flags().BoolVar(&opts.CheckInstall, "install", true, "Also check if app is installed")
```

Comment:
I find `--install` confusing as it might give the idea that something will be installed.
I would rather use `--check-installation`, unless it is too long.

Reply:
I think it is too long. I read it as do "ps --install" command that checks running apps along with their installation. Also, it is in sync what we already have for "check" command.